internal/oidc/client.go

```diff
@@ -73,3 +125,64 @@ func (c InternalClient) IsPublic() bool {
 func (c InternalClient) GetAudience() fosite.Arguments {
 	return c.Audience
 }
+
+// GetResponseModes returns the valid response modes for this client.
+//
+// Implements part of fosite.OpenIDConnectClient.
+func (c InternalClient) GetResponseModes() []fosite.ResponseModeType {
+	return c.ResponseModes
+}
+
+// GetRequestURIs returns the valid request URIs for this client.
+//
+// Implements part of fosite.OpenIDConnectClient.
+func (c InternalClient) GetRequestURIs() []string {
```

Comment:
Should be in a dedicated PR as discussed.

Reply:
Removed.